index.bs

```diff
@@ -1384,22 +1384,25 @@ run these steps:
 1. Let |accepts options| be a empty [=/list=] of [=pairs=].
 1. [=list/For each=] |type| of |options|.{{FilePickerOptions/types}}:
   1. Let |description| be |type|.{{FilePickerAcceptType/description}}.
-  1. [=map/For each=] |type string| → |extensions| of |type|.{{FilePickerAcceptType/accept}}:
+  1. [=map/For each=] |type string| → |suffixes| of |type|.{{FilePickerAcceptType/accept}}:
     1. Let |parsedType| be the result of [=parse a MIME type=] with |typeString|.
     1. If |parsedType| is failure, throw a {{TypeError}}.
     1. If |parsedType|'s [=MIME type/parameters=] are not empty, throw a {{TypeError}}.
+    1. If |suffixes| is a string:
+      1. If |suffixes| does not start with ".", throw a {{TypeError}}.
+    1. Otherwise, [=list/for each=] |suffix| of |suffixes|:
+      1. If |suffix| does not start with ".", throw a {{TypeError}}.
 
   1. Let |filter| be the following steps, given a |filename| (a [=string=]), and a |type| (a [=MIME type=]):
-    1. [=map/For each=] |type string| → |extensions| of |type|.{{FilePickerAcceptType/accept}}:
+    1. [=map/For each=] |type string| → |suffixes| of |type|.{{FilePickerAcceptType/accept}}:
     1. Let |parsedType| be the result of [=parse a MIME type=] with |typeString|.
```

Comment:
Same _type string_ vs. _typeString_ inconsistency here.

Reply:
hah, by making the same mistake twice I fooled bikeshed's unused variable warning...